app/models/concerns/decidim/ludens/organization_model_extend.rb

```diff
@@ -52,6 +52,17 @@ def step_scores
         def toggle_ludens
           update!(enable_ludens: !enable_ludens)
         end
+
+        def last_done_recommendation
+          last = assistant["last"]
+          ParticipativeAction.find_by(id: last)
```

Comment:
The variable initialization is not necessary here
```suggestion
          ParticipativeAction.find_by(id: assistant["last"])
```